resources/XZ_Supply_chain_attack.md

```diff
@@ -0,0 +1,108 @@
+# CMS DSAC OSPO REPORT: CVE-2024-3094: Backdoor in xz
+
+## What happened?
+
+A supply chain attack, or a cyber attack aimed at breaking the weakest part of a software development process, on an important open source archival tool was discovered on March 29th, 2024. A backdoor, or alternative method of access, was added to some more recent versions of the xz tool that allowed an attacker to do RCE (Remote Code Execution) on specific machines that have ssh enabled. XZ is an immensely popular software that is used in file compression broadly.
+
+Luckily, this attack did not affect many systems as it was caught before the changes were merged into their intended targets. Linux works as a grouping of specific programs and the specific groupings are called Linux distributions. The attack targeted specific distributions of Linux, an open source operating system, that used Debian or Red Hat style software packaging. The attack only made it into unstable versions of Debian and Fedora and were rolled back very quickly. The vulnerability was caught by a Microsoft employee that noticed that their ssh login attempts were taking longer than expected. SSH is a tool used for securely accessing computer systems remotely.
+
+On further investigation of the person responsible for the attack, who goes by the alias ‘Jia Tan’, it seems that they showed a pattern of introducing compromising open source contributions into key projects. This person, or very likely a group of people with considerable resources, has spent years gaining the trust of key open source developers and projects. A key and most concerning example of a malicious contribution that ‘Jia Tan’ made was to the libarchive repository to change instances of a safe call to a function to an unsafe version of that function (printf). This persona has not shown activity since the attack was discovered.
+
+## What does this mean for CMS systems?
+
+This attack should not affect us in any major way. Any crucial systems that CMS is responsible for should not have been running experimental versions of Debian or Fedora using systemd services, and that is the only way key systems would be affected. Systems that do not use Systemd services, such as MacOS, would not be affected. Systemd is a system processing monitor and suite of tools that is only available for the Linux operating system.
+
+Going forward, CMS should encourage the adoption of healthy standards such as:
+
+- Code review
+- Applying security scanning tools
+- Isolation of build environments
+- Ensuring that crucial systems maintained by contractors show compliance
+
+This is part of CISA’s secure by design initiative that is extremely relevant to this attack. Additionally, it is important to encourage the continued adoption of the CMS OIT Cybersecurity initiatives such as CMS (GRC) Governance Risk and Compliance.
```

Comment:
IDEA: Link to CISA Secure By Design Initiative: https://www.cisa.gov/securebydesign